### PR TITLE
Luke-Skywalker

### PR DESCRIPTION
**Pull request by asalcedo07**
Add  Luke Skywalker interview in a post-Force Awakens world.

### CODE REVIEW

**Comment by asalcedo07** [comment]
Please review this, @casey-collab

I hope everything works.

File: english/luke-skywalker/luke-skywalker.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+So you have an interview with Luke Skywalker.
+What do you ask him?
+
+[Are you Rey's Father?](starwars8/wait-for-8.md)
```
Comment (casey-collab) [inline english/luke-skywalker/luke-skywalker.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/luke-skywalker/luke-skywalker.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+So you have an interview with Luke Skywalker.
```
Comment (casey-collab) [inline english/luke-skywalker/luke-skywalker.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

File: english/luke-skywalker/starwars8/Wait-for-8.md [hunk 3]
```diff
@@ -0,0 +1,2 @@
+"yea, you know I can't say anything.
```
Comment (casey-collab) [inline english/luke-skywalker/starwars8/Wait-for-8.md]: This page is also unreachable.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by asalcedo07** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :seedling:

**Pull request merged**